zips/draft-jackgavigan-defer-and-extend.rst

```diff
@@ -0,0 +1,186 @@
+August 2021 indicated that the Zcash community believed that ZSAs should be 
+first priority for Zcash following the activation of NU5. [#priorities]_
+
+In January 2022, ZCG provided funding for QEDIT to research and implement ZSAs 
+[#zsagrant]_, and the functionality was demonstrated using a fork of zcashd 
+during Zcon4 in August 2023. [#zsademo]
+
+Due to ECC's decision to no longer support new protocol features in the zcashd 
+node implementation, zcashd must be deprecated before ZSAs can activate on 
+Mainnet. Zcashd deprecation was expected to be complete by April 2025. However, 
+the necessary work was not prioritized. [#nu7-timeline]_ As a result, as of March 
+2025, there is no confirmed schedule for zcashd deprecation (and, therefore, no 
+schedule for the activation of ZSAs on Mainnet). 
+
+The activation of ZSAs on Mainnet is also dependent on the merging of necessary 
+code changes into the core Zcash cryptographic libraries and the new Zcash stack 
+(Zebra, Zaino and Zallet) that will replace zcashd. With limited resources, 
+anything that distracts the engineers who are working on the new stack is likely 
+to delay ZSA activation further. 
+
+In addition, any changes to the core Zcash cryptographic libraries or Zebra -- 
+including the definition and implementation of a Lockbox distribution mechanism 
+-- would require that QEDIT rebase their code changes, incurring further cost 
+and delay. 
+
+This ZIP therefore proposes expressly prohibiting any disbursement of funds from 
+the Lockbox until after Zcash Shielded Assets is active on Mainnet. 
+
+It is anticipated that the majority of teams currently working on the development 
+or outreach required to deprecate zcashd, and activate ZSAs, will require funding 
+in order to see this effort through to a successful conclusion. Following the end 
+of direct funding of BP and ZF, ZCG is the sole source of funding in the Zcash 
+ecosystem. 
+
+Under the existing consensus rules, after block height 3146400, 100% of the 
+block subsidies would be allocated to miners, and no further funds would be 
+automatically allocated to the Lockbox or to ZCG. Consequently, unless the 
+community takes action to approve new block-subsidy-based funding, existing 
+teams dedicated to development or outreach or furthering charitable, 
+educational, or scientific purposes would likely need to seek other sources of 
+funding; failure to obtain such funding would likely impair their ability to 
+continue serving the Zcash ecosystem.
+
+This ZIP therefore proposes extending the Current Dev Fund by one year, to 
+ensure that ZCG receives sufficient funding to enable it to fund teams that are 
+carrying out the work required to deprecate zcashd, and activate ZSAs. 
+
+
+Requirements
+============
+
+This ZIP is dependent on ZIP 1015, [#zip-1015]_ and ZIP 2001. [#zip-2001]_ 
+Amendments to these ZIPs are specified below. 
+
+
+Non-requirements
+================
+
```

Comment:
A non-requirement of this ZIP could be not defining or supporting any disbursement mechanism after ZSA deployment